db/migrate/20210627155647_add_call_leg_to_call_data_records.rb

```diff
@@ -0,0 +1,6 @@
+class AddCallLegToCallDataRecords < ActiveRecord::Migration[6.1]
+  def change
+    add_column :call_data_records, :call_leg, :string, default: "A", null: false
+    add_index :call_data_records, :call_leg, where: "call_leg = 'A'"
```

Comment:
I'm not sure we need the where clause here? Most will be leg A but we don't find by call_leg at the moment. Maybe we can have an index on the leg, don't need the where

Reply:
I think we may need it because we have `has_one :call_data_record, -> { where(call_leg: 'A') }` if we remove where from has_one, we might return the wrong one (call_leg_B).

Reply:
oh but at most it will only return 2 CDRs? We definitely need the `where` in the `has_one` but don't need it in the migration